packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts

```diff
@@ -130,16 +126,15 @@ class OidcConfigBuilder {
     demonstratingProofOfPossessionJwkJson: null,
     demonstratingProofOfPossessionConfiguration: null,
     demonstratingProofOfPossessionOnlyWhenDpopHeaderPresent: false,
-    allowMultiTabLogin: true,
   };
 
   public withTestingDefault(): OidcConfigBuilder {
     this.oidcConfig.configurationName = 'test';
     this.oidcConfig.tokens = new TokenBuilder().withNonExpiredToken().build();
     this.oidcConfig.status = 'NOT_CONNECTED';
-    this.oidcConfig.state = { tab1: 'state' };
```

Comment:
I think the correct new version is { tab1: 'state' }; in the whole file @wermanoid

Reply:
I think it explain why unit tests are failing @wermanoid

Reply:
Ok, updated from clear main branch. Should not introduce anything unexpected now